src/main/java/com/avispl/symphony/dal/infrastructure/management/samsung/smartthings/SamsungSmartThingsAggregatorCommunicator.java

```diff
@@ -1973,7 +1977,7 @@ private void populateCommonDetailViewControllableProperties(Map<String, String>
 						.map(Slider::getRange);
 
 				if (range.isPresent() && range.get().size() == 2) {
-					if (StringUtils.isNullOrEmpty(detailViewPresentation.getSlider().getValue())) {
+					if (StringUtils.isNullOrEmpty(detailViewPresentation.getSlider().getValue()) || SmartThingsConstant.NULL.equalsIgnoreCase(detailViewPresentation.getSlider().getValue())) {
```

Comment:
can Slider be null here? 
Also, further on the line 1983  detailViewPresentation.getSlider().getValue() is referenced again